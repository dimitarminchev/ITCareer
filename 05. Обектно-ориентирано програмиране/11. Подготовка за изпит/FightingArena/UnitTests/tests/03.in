CreateHero:Warrior:Pesho
Fight:Gosho:Pesho
Fight:Pesho:Gosho
CreateHero:Assassin:Gosho
Fight:Gosho:Pesho
CreateWeapon:Gosho:Sword:WeaponName:5:5:5
Fight:Gosho:Pesho
CloseArena