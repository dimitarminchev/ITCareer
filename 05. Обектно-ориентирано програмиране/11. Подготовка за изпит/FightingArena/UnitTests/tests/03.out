Warrior: Pesho joined the Arena!
Hero with name: Gosho does not exist!
Hero with name: Gosho does not exist!
Assassin: Gosho joined the Arena!
No winner in battle between Gosho and Pesho!
Successfully armed hero Gosho with weapon Sword!
Winner in the battle between Gosho and Pesho is Gosho with 60.00.
Heroes: 2
Assassin: Gosho - Level: 0
Experience: 45
Sword: WeaponName
  *Strength: 20
  *Agility: 5
  *Intelligence: 5
Power: 60.00
Warrior: Pesho - Level: 0
Experience: 15
Power: 0.00